discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -736,14 +737,27 @@ def _configure_load_balancer(self, controller):
         lb_controller = LoadBalancerController(tf)
         lb_controller.set_load_balancing_config(load_balancer_ip, master_ips, worker_ips)
 
+    @classmethod
+    def _get_namespace_index(cls, libvirt_network_if):
+        # Hack to retrieve namespace index - does not exist in tests
+        matcher = re.match(r'^tt(\d+)$', libvirt_network_if)
+        return int(matcher.groups()[0]) if matcher is not None else 0
+
     @classmethod
     def _configure_nat(cls, controller):
```

Comment:
Maybe set it as public so we could create fixture on BaseTest

Reply:
OK